app/views/posts/show.html.erb

```diff
@@ -0,0 +1,85 @@
+<div class="post-image">
+  <% if @post.image.present? %>
+    <%= image_tag @post.image.url(:medium) %>
+  <% end %>
+</div>
+
+<div class="post-title">
+  <h1><%= @post.title %></h1>
+  <p>
+    by: <%= @post.creator_user_name %>
+  </p>
+</div>
+<hr>
+<% if @post.tags.any? %>
+  <% @post.tags.each do |tag| %>
+  &nbsp
+    <div class="label label-default">
+      <%= tag.name %>
+    </div>
+  <% end %>
+<% end %>
+&nbsp
+<% if((user_signed_in?) && (can? :create, @post.favourites.new))  %>
+  <div class="fav">
+    <% if @post.fav_by current_user %>
+       <%= link_to fa_icon("star"), [@post, @post.favourite_for(current_user)], method: :delete %>
+      <%= @post.favs_count %>, including you.
+    <% else %>
+      <%= link_to fa_icon("star-o"), post_favourites_path(@post), method: :post%>
+      <%= @post.favs_count %>.
+    <% end %>
+
+  </div>
+<% elsif(user_signed_in?) && !(can? :create, @post.favourites.new) %>
+  <div class="fav">
+    <% if @post.fav_by current_user %>
+       <%= fa_icon("star")%>
+      <%= @post.favs_count %>, including you.
+    <% else %>
+      <%= fa_icon("star-o")%>
+      <%= @post.favs_count %>.
+    <% end %>
+
+  </div>
+<% end %>
+
+<div class="">
+  <p class="post-body">
+    <%= @post.body %>
+  </p>
+  <div class="align-right indie-flower">
+    <% if can? :edit, @post %>
+    <%= link_to "edit", edit_post_path(@post) %> &nbsp&nbsp&nbsp
+    <% end %>
```

Comment:
` &nbsp&nbsp&nbsp` whats happening here, you trying to indent? uses some padding instead